### ISSUE TITLE
Selecting a .wav file in the Assets Browser crashes the engine

### ISSUE THREAD

**Issue by yuberee** [Facepunch/sbox-issues#1994]
### Describe the bug

Whenever I'm selecting a .wav file in the new Assets Browser, the sound will play and as soon as it ends the engine crashes.
Unsure if it happens with other sound file formats, I have not tested with them.

### To Reproduce

1. Clone the repo: https://github.com/yuberee/xoxoxo
2. Open the NEW Assets Browser
3. Navigate to the Sounds folder
4. Select one of the .wav files

### Expected behavior

Not to crash :-)

### Additional context

- No dump file gets generated
- All the sound files in the repository are compressed (Microsoft ADPCM), single channel, 44.1Mhz; Uncompressed .wav files still crash the engine
- This is the last thing the console log show: 
```
2022/07/02 18:45:13.352	AssetSystem	On-demand recompile of asset sounds/grunt1.vsnd	
2022/07/02 18:45:13.352	AssetSystem	 -> reason: At least one out-of-date dependency: sounds/humans/grunt1.wav (file stopped existing)	
```
Which is untrue and doesn't seem to print every single time

**Comment by aylaylay** [comment]
Should be good now

**Comment by yuberee** [comment]
I'm not getting any more crashes.
Coincidentally, this seems to also fix the crashes described in issue #1069, since it's understandable for the engine to not include multiple compression codecs I'll go ahead and close that issue as well.
